### PR TITLE
docs: Fix broker class in create command

### LINKED ISSUE #1540: `kn broker create` example has incorrect flag
`--broker` should be `--class`
https://github.com/knative/client/blob/883ff7d4f4d117d1b688626862d11179ebc20237/pkg/kn/commands/broker/create.go#L34

### PR DESCRIPTION
**Pull request by embano1**
## Description

Fix `kn broker create` example docs.

## Changes

* Change `--broker` to `--class`

## Reference

Fixes #1540

### CODE REVIEW

File: pkg/kn/commands/broker/create.go [hunk 1]
```diff
@@ -31,7 +31,7 @@ var createExample = `
   kn broker create mybroker
 
   # Create a broker 'mybroker' in the 'myproject' namespace and with a broker class of 'Kafka'
-  kn broker create mybroker --namespace myproject --broker Kafka
```
Comment (navidshaikh) [inline pkg/kn/commands/broker/create.go:34]: Run `./hack/build.sh` locally to generate the [docs](https://github.com/knative/client/blob/main/docs/cmd/kn_broker_create.md) and commit the generated changes in this PR, the build tests are complaining absence of (auto) generated docs

Comment (embano1) [reply pkg/kn/commands/broker/create.go:34]: Done.

**Review by navidshaikh — APPROVED** [review]

**Pull request merged**